lms/djangoapps/instructor_task/models.py

```diff
@@ -315,9 +315,9 @@ def store_rows(self, course_id, filename, rows):
         if six.PY2:
             output_buffer.write(codecs.BOM_UTF8)
         csvwriter = csv.writer(output_buffer)
-        csvwriter.writerows(self._get_utf8_encoded_rows(rows))
+        csvwriter.writerows(self._get_utf8_encoded_rows(rows, encode_for_utf8=True))
         output_buffer.seek(0)
-        self.store(course_id, filename, output_buffer)
+        self.store(course_id, filename, output_buffer, batched=True)
 
```

Comment:
The `store_rows ` usage of the instructor_task app is widespread as it's being used in different xblocks as well as edx itself. So It's good to do the change where it's impacting and use `add_rows` and call `store` method rather than changing in a core function.

Reply:
@mudassir-hafeez 
I have updated the Pull Request and used default arguments in `store_rows` so that the widespread usage of this method doesn't get affected.